README.md

```diff
@@ -14,30 +14,76 @@ This README contains a high level description of contributing a SWIP, for conven
  3. Add your SWIP to your fork of the repository. There is a [template SWIP here](SWIPs/swip-X.md).
  4. Submit a Pull Request to Swarm's [SWIPs repository](https://github.com/ethersphere/SWIPs).
 
-Your first PR should be a first draft of the final SWIP. It must meet the formatting criteria. An editor will manually review the 
-first PR for a new SWIP and assign it a number before merging it. Make sure you include a `discussions-to` header with the URL to 
-a discussion forum or open GitHub issue where people can discuss the SWIP as a whole.
+A SWIP proposal or suggested edits (or updates) may come from anyone (inside or outside the organisation) and are received in the form of a pull request to the [SWIP repository](https://github.com/ethersphere/SWIPs). Throughout the lifetime of a SWIP from proposal to final accepted, we use the PR-review infrastructure of github to comment and improve the SWIP.
 
-If your SWIP requires images, the image files should be included in a subdirectory of the `assets` folder for that SWIP as 
-follows: `assets/swip-X` (for swip **X**). When linking to an image in the SWIP, use relative links such as 
-`../assets/swip-X/image.png`.
+THe github permissions of the repo reflect these privileges:
+- Anyone can open a new PR.
+- Only SWIP editors can merge a PR to master.
 
-When you believe your SWIP is mature and ready to progress past the draft phase, you should open a PR changing the state of your 
-SWIP to 'Final'. An editor will review your draft and ask if anyone objects to its being finalised. If the editor decides there 
-is no rough consensus - for instance, because contributors point out significant issues with the SWIP - they may close the PR and 
-request that you fix the issues in the draft before trying again.
+### Creating a new SWIP
 
-For a detailed description of the SWIP process see [SWIP-0](SWIPs/swip-0.md).
+A new SWIP can be proposed by opening a PR in the [SWIP repo](https://github.com/ethersphere/SWIPs) titled `SWIP-<XXXX>: <title>` where `XXXX` is the SWIP number left padded with zeros up to four digits (i.e., fprinted with the format string `%04d`). 
```

Comment:
To avoid clashes on SWIP numbers, let us define that a SWIP must have the same number as the PR. 

Also, I would be against left padding with zeros to make it for digits, as I see no need for that. The number could quickly grow to more than four digits. And also, the practice in Ethereum is there is no padding (example https://eips.ethereum.org/all).

Reply:
OK then. Then it is no longer relevant to merge the claim PR early on, it simplifies the process.

The need for XXXX numbering is solely the neat temporally correct listing of SWIP md files

Reply:
@crtahlin How about we don't use /assets/SWIP-XXXX/ subdirectories, rather a global assets directory to enable shared figures?

Reply:
I would say that each SWIP should have their own folder of assets for easier reference and browsing within the SWIP. If in the future we see the need for having shared assets among the SWIPs, we can have a global one.